README.md

```diff
@@ -1,9 +1,100 @@
-This is a [demo app](https://web.dev/porting-libusb-to-webusb/) running gPhoto2 on the Web:
-
-![A picture of DSLR camera connected via a USB cable to a laptop. The laptop is running the Web demo mentioned in the article, which mirrors a live video feed from the camera as well as allows to tweak its settings via form controls.](https://web-dev.imgix.net/image/9oK23mr86lhFOwKaoYZ4EySNFp02/MR4YGRvl0Z9AWT6vv3sQ.jpg?auto=format&w=1600)
+# Web-gPhoto2
+A gPhoto2 implementation using WebAssembly to control DSLR cameras from the browser.
 
 Powered by a [custom fork](https://github.com/RReverser/libgphoto2) of [libgphoto2](https://github.com/gphoto/libgphoto2), the [WebUSB](https://github.com/WICG/webusb) backend of [libusb](https://github.com/libusb/libusb), and WebAssembly via [Emscripten](https://emscripten.org/).
 
+# NPM
+## Installation
+```bash
+npm install web-gphoto2
+// or
+yarn add web-gphoto2
+```
+
+## Usage
+
+A short example on how to use this package:
+```ts
+import Camera from 'web-gphoto2';
+
+let camera = new Camera();
+
+async function connectCamera() {
+    // @ts-ignore
+    await navigator.usb.requestDevice({
```

Comment:
I don't remember, but this is the generic selector for DSLR cameras, right? If so, this call should probably also live in the module like `showCameraPicker` or something so that users don't have to remember or copy-paste those values.